types/luxon/index.d.ts

```diff
@@ -0,0 +1,365 @@
+            static TIME_24_WITH_SHORT_OFFSET: DateTimeFormat;
+            static TIME_SIMPLE: DateTimeFormat;
+            static TIME_WITH_LONG_OFFSET: DateTimeFormat;
+            static TIME_WITH_SECONDS: DateTimeFormat;
+            static TIME_WITH_SHORT_OFFSET: DateTimeFormat;
+            static fromHTTP(text: string, options?: DateTimeOptions): DateTime;
+            static fromISO(text: string, options?: DateTimeOptions): DateTime;
+            static fromJSDate(
+                date: Date,
+                options?: DateTimeJSOptions
+            ): DateTime;
+            static fromMillis(ms: number, options?: DateTimeOptions): DateTime;
+            static fromObject(obj: DateObject): DateTime;
+            static fromRFC2822(
+                text: string,
+                options?: DateTimeOptions
+            ): DateTime;
+            static fromString(
+                text: string,
+                format: string,
+                options?: DateTimeOptions
+            ): DateTime;
+            static fromStringExplain(
+                text: string,
+                format: string,
+                options?: DateTimeOptions
+            ): Object;
+            static invalid(reason: any): DateTime;
+            static local(
+                year?: number,
+                month?: number,
+                day?: number,
+                hour?: number,
+                minute?: number,
+                second?: number,
+                millisecond?: number
+            ): DateTime;
+            static max(...dateTimes: DateTime[]): DateTime | undefined;
+            static min(...dateTimes: DateTime[]): DateTime;
+            static utc(
+                year?: number,
+                month?: number,
+                day?: number,
+                hour?: number,
+                minute?: number,
+                second?: number,
+                millisecond?: number
+            ): DateTime;
+            day: number;
+            daysInMonth: number;
+            daysInYear: number;
+            hour: number;
+            invalidReason: string;
+            isInDST: boolean;
+            isOffsetFixed: boolean;
+            isValid: boolean;
+            locale: string;
+            millisecond: number;
+            minute: number;
+            month: number;
+            monthLong: string;
+            monthShort: string;
+            numberingSystem: string;
+            offset: number;
+            offsetNameLong: string;
+            offsetNameShort: string;
+            ordinal: number | DateTime;
+            outputCalendar: string;
+            second: number;
+            weekNumber: number;
+            weekYear: number;
+            weekday: string;
+            weekdayLong: string;
+            weekdayShort: string;
+            year: number;
+            zoneName: string;
+            diff(
+                other: DateTime,
+                unit: string | string[],
```

Comment:
I thiiiink `unit` [is optional](https://moment.github.io/luxon/docs/class/src/datetime.js~DateTime.html#instance-method-diff).